giraffe/src/utils/geo.test.ts

```diff
@@ -0,0 +1,126 @@
+import {InfluxColors} from '../constants/colorSchemes'
+import {
+  GeoCircleViewLayer,
+  GeoPointMapViewLayer,
+  GeoQueryVariables,
+} from '../types/geo'
+import {
+  calculateVariableAssignment,
+  formatCircleMarkerRowInfo,
+  formatPointLayerRowInfo,
+  getMinZoom,
+  getRowLimit,
+} from './geo'
+import {preprocessData} from '../components/geo/processing/tableProcessing'
+import {fromFlux} from './fromFlux'
+import {geoCSV} from '../../../stories/src/data/geo'
+
+const pointMapLayer: GeoPointMapViewLayer = {
+  type: 'pointMap',
+  colorDimension: {label: 'Value'},
+  colorField: '_value',
+  colors: [
+    {type: 'min', hex: InfluxColors.Star},
+    {value: 50, hex: InfluxColors.Star},
+    {type: 'max', hex: InfluxColors.Star},
+  ],
+  isClustered: false,
+}
+
+const circleMapLayer: GeoCircleViewLayer = {
+  type: 'circleMap',
+  radiusField: 'magnitude',
+  radiusDimension: {label: 'Magnitude'},
+  colorDimension: {label: 'Duration'},
+  colorField: 'duration',
+  colors: [
+    {type: 'min', hex: '#ff00b3'},
+    {value: 50, hex: '#343aeb'},
+    {type: 'max', hex: '#343aeb'},
+  ],
+}
+
+describe('getMinZoom', () => {
+  it('can return minimum zoom for viewing maps', () => {
```

Comment:
are there more zoom mappings to width? or is 500 px wide => level 1 zoom the only one there is?

Reply:
I am not sure, I don't see width being used anywhere in the ui or in the storybook. I have added the test so that if the formula to calculate minZoom changes, it will get caught here.